flytekit/tools/fast_registration.py

```diff
@@ -58,28 +100,74 @@ def fast_package(
         ignores = default_ignores
     ignore = IgnoreGroup(source, ignores)
 
+    # Remove this after original tar command is removed.
```

Comment:
Why are we keeping the original tar command? Is it for backward compatibility?

Reply:
yup.